java.html.markdown

```diff
@@ -686,7 +686,6 @@ public abstract class Mammal()
         return true;
     }
 }
-```
```

Comment:
You should not remove this, as it closes the markdown formatting for code (started on line 18)